pkg/config/config.go

```diff
@@ -217,13 +242,17 @@ func newChannelGroup(channelConfig Channel) (*cb.ConfigGroup, error) {
 		if err != nil {
 			return nil, err
 		}
+	} else {
+		return nil, errors.New("consortium is not defined in channel config")
```

Comment:
given you return within else, might make sense to revert logic of the `if-then-else` to save on indentation and return on error case immediately?

Reply:
Thank you very much for your suggestion. I have reverted the logic.